src/controllers/tests/help.controller.test.ts

```diff
@@ -0,0 +1,132 @@
+            expect(response.body).toEqual({
+                "command": "/generate",
+                "method": "POST",
+                "summary": "Generate the given AsyncAPI template.",
+                "requestBody": {
+                    "type": "object",
+                    "required": [
+                        "asyncapi",
+                        "template"
+                    ],
+                    "properties": {
+                        "asyncapi": {
+                            "$ref": "https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#asyncapi-object"
+                        },
+                        "template": {
+                            "type": "string",
+                            "description": "Template name to be generated.",
+                            "enum": [
+                                "@asyncapi/dotnet-nats-template",
+                                "@asyncapi/go-watermill-template",
+                                "@asyncapi/html-template",
+                                "@asyncapi/java-spring-cloud-stream-template",
+                                "@asyncapi/java-spring-template",
+                                "@asyncapi/java-template",
+                                "@asyncapi/markdown-template",
+                                "@asyncapi/nodejs-template",
+                                "@asyncapi/nodejs-ws-template",
+                                "@asyncapi/python-paho-template",
+                                "@asyncapi/ts-nats-template"
+                            ]
+                        },
+                        "parameters": {
+                            "type": "object",
+                            "description": "Template parameters to be generated. Each template has different parameters that you should check in the documentation, \nwhich is usually located in the template's repository.\nThis field is optional but may be required for some templates.\n",
+                            "additionalProperties": true
+                        }
+                    }
+                }
+            });
+        });
+
+        it('should return 404 error for an invalid command', async () => {
+            const response = await request(app.getServer())
+                .get('/v1/help/invalidCommand')
+                .expect(404);
+
+            expect(response.body.message).toBe('Failed to get help. The given AsyncAPI command is not valid.');
+        });
+
+        it('should return 404 error for a command without a method', async () => {
+            
+            (fetchCommands as jest.Mock).mockResolvedValue({
+                paths: {
+                    "/someCommand": {}
+                }
+            });
+
+            const response = await request(app.getServer())
+                .get('/v1/help/someCommand')
+                .expect(404);
+
+            expect(response.body.message).toBe('Failed to get help. The given AsyncAPI command is not valid.');
```

Comment:
@princerajpoot20  Can you refactor so that the `404` response is always using the Problem schema: `$ref: '#/components/schemas/Problem'`? You can see an example of how to use this schema in controllers in `generate.controller` - 
https://github.com/asyncapi/server-api/blob/ea68f4963b7327562aedda0a088208e3a9b6b406/src/controllers/generate.controller.ts#L86C32-L86C32